src/api/fungibleAsset.ts

```diff
@@ -125,4 +122,27 @@ export class FungibleAsset {
     });
     return getCurrentFungibleAssetBalances({ aptosConfig: this.config, ...args });
   }
+
+  /**
+   *  Transfer `amount` of fungible asset from sender's primary store to recipient's primary store.
+   *
+   * Use this method to transfer any fungible asset including fungible token.
+   *
+   * @param sender The sender account
+   * @param fungibleAssetMetadataAddress The fungible asset account address.
+   * For example if you’re transferring USDT this would be the USDT address
+   * @param recipient The recipient account address
+   * @param amount Number of assets to transfer
+   *
+   * @returns A SingleSignerTransaction that can be simulated or submitted to chain.
+   */
+  async transferFungibleAsset(args: {
+    sender: Account;
+    fungibleAssetMetadataAddress: AccountAddress;
+    recipient: AccountAddress;
+    amount: AnyNumber;
+    options?: InputGenerateTransactionOptions;
```

Comment:
We could probably do something fancy in the future, which determines whether it's a primary store, and does that, or if it's a non-primary store.